network/client.go

```diff
@@ -44,6 +44,7 @@ func (cl *Client) Send(dst *ServerIdentity, msg Body) (*Packet, error) {
 	baseIDLock.Unlock()
 	sid := NewServerIdentity(kp.Public, NewAddress(dst.Address.ConnType(),
 		"client:"+strconv.FormatUint(id, 10)))
+	sid.Cert = dst.Cert
```

Comment:
? Is it necessary ? The authentication is done only one-way right ? (i.e. the connectOR knows the certificate of the connectEE but not the inverse)